sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -45,6 +46,7 @@ public partial class ProcessMessageEventArgs : System.EventArgs
     public partial class ProcessorReceiveActions
     {
         protected ProcessorReceiveActions() { }
+        public virtual System.Threading.CancellationToken GetLockExpiryCancellationToken(Azure.Messaging.ServiceBus.ServiceBusReceivedMessage receivedMessage) { throw null; }
```

Comment:
Hmm what is the use case for this method?

Reply:
When you use the receive actions to fetch more messages you can get a dedicated cancellation token representing the LockUntil time period for every tracked message. Happy to remove if you think that is overkill.

Reply:
I would leave this out of scope. The receive actions is a very niche feature. We could always add something like this in the future.